### PR TITLE
Use prefixed node: APIs

### LINKED ISSUE #113: Could not resolve "path" when building for Cloudflare Workers
Using:

```ts
pnpm vite build
pnpm wrangler build
```

```
 ⛅️ wrangler 3.62.0
-------------------


✘ [ERROR] Could not resolve "path"

    node_modules/.pnpm/telefunc@0.1.74_@babel+core@7.23.9_@babel+parser@7.24.5_@babel+types@7.24.5_react-streaming@0_3t7m6eqtjjvdb6ys45f7c5hbhi/node_modules/telefunc/dist/cjs/node/server/runTelefunc/loadTelefuncFilesFromConfig.js:8:23:
      8 │ const path_1 = require("path");
        ╵                        ~~~~~~

  The package "path" wasn't found on the file system but is built into node.
  Add "node_compat = true" to your wrangler.toml file and make sure to prefix the module name with "node:" to enable Node.js compatibility.


✘ [ERROR] Could not resolve "path"

    node_modules/.pnpm/telefunc@0.1.74_@babel+core@7.23.9_@babel+parser@7.24.5_@babel+types@7.24.5_react-streaming@0_3t7m6eqtjjvdb6ys45f7c5hbhi/node_modules/telefunc/dist/cjs/node/server/serverConfig.js:4:23:
      4 │ const path_1 = require("path");
        ╵                        ~~~~~~

  The package "path" wasn't found on the file system but is built into node.
  Add "node_compat = true" to your wrangler.toml file and make sure to prefix the module name with "node:" to enable Node.js compatibility.


✘ [ERROR] Could not resolve "path"

    node_modules/.pnpm/telefunc@0.1.74_@babel+core@7.23.9_@babel+parser@7.24.5_@babel+types@7.24.5_react-streaming@0_3t7m6eqtjjvdb6ys45f7c5hbhi/node_modules/telefunc/dist/cjs/utils/getOutDirs.js:9:39:
      9 │ const path_1 = __importDefault(require("path"));
        ╵                                        ~~~~~~

  The package "path" wasn't found on the file system but is built into node.
  Add "node_compat = true" to your wrangler.toml file and make sure to prefix the module name with "node:" to enable Node.js compatibility.
```

I use:

```json
vike 0.4.177
vite 5.2.11
```

Possible solution is to replace all node imports to prefixed. When I actually replaced in `node_modules` it works correctly

### PR DESCRIPTION
**Pull request by sergeysova**
fixes #113

### CODE REVIEW

**Comment by brillout** [comment]
:+1: Also, one thing we could do is to add a Biome rule to avoid mistakenly omitting the `node:` prefix. WDYT?

File: biome.json [hunk 1]
```diff
@@ -16,7 +16,13 @@
     }
   },
   "linter": {
-    "enabled": false
+    "enabled": true,
+    "rules": {
+      "recommended": false,
```
Comment (sergeysova) [inline biome.json:21]: Disabled recommended rules due to much errors, fixing is out of scope of this PR.

```
The number of diagnostics exceeds the number allowed by Biome.
Diagnostics not shown: 164.
Checked 125 files in 25ms. No fixes applied.
Found 142 errors.
Found 1 warning.
```

**Comment by brillout** [comment]
Very nice.

Running `$ biome lint --write .` doesn't fix the imports that are missing the `node:` prefix, can we make Biome autofix this as well? It says `Unsafe fix` so maybe that's why, and maybe we need to tell Biome to apply its autofix despite being "unsafe".

Biome doesn't complain about dynamic imports such as `import('path')`, is there a way to make it complain about dynamic imports as well?

**Comment by brillout** [comment]
The arg `--unsafe` does the trick.

**Comment by brillout** [comment]
Closing in favor of https://github.com/brillout/telefunc/pull/115.

**Pull request closed without merging**